pkg/summarizer/summary.go

```diff
@@ -592,6 +608,38 @@ func latestGreen(grid *statepb.Grid, useFirstExtra bool) string {
 	return noGreens
 }
 
+func getHealthinessForInterval(grid *statepb.Grid, tabName string, timeNow clock, interval int) *summarypb.HealthinessInfo {
+	analyzer := naiveanalyzer.NaiveAnalyzer{}
+
+	currentTime := timeNow()
+	now := goBackDays(0, currentTime)
+	oneInterval := goBackDays(interval, currentTime)
+	twoIntervals := goBackDays(2*interval, currentTime)
+
+	healthiness := CalculateHealthiness(grid, &analyzer, oneInterval, now, tabName)
+	pastHealthiness := CalculateHealthiness(grid, &analyzer, twoIntervals, oneInterval, tabName)
+	CalculateTrend(healthiness, pastHealthiness)
+
+	return healthiness
+}
+
+func goBackDays(days int, currentTime time.Time) int {
+	// The primary usage of goBackDays is to get the time intervals for our flakiness report
```

Comment:
nit: This comment can read more like the one for CalculateTrend ("gobackDays gets...")